frontend/test/metabase/scenarios/admin/databases/edit.cy.spec.js

```diff
@@ -95,17 +95,15 @@ describe("scenarios > admin > databases > edit", () => {
         cy.wait("@databaseUpdate").then(({ request, response }) => {
           expect(request.body.cache_ttl).to.equal(32);
           expect(response.body.cache_ttl).to.equal(32);
+        });
```

Comment:
Not sure why was the rest of this test scoped under the first `wait` block?
It can (and should) live independently of it, so that's what I did here.